src/main/java/de/tum/in/www1/exerciseapp/domain/DropLocation.java

```diff
@@ -43,7 +43,7 @@
     @JsonIgnore
     private DragAndDropQuestion question;
 
-    @OneToMany(cascade = CascadeType.REMOVE, fetch = FetchType.EAGER, orphanRemoval = true, mappedBy = "dropLocation")
+    @OneToMany(cascade = CascadeType.REMOVE, orphanRemoval = true, mappedBy = "dropLocation")
```

Comment:
@MoritzIssig: Did you talk about this change with @VSchlattinger 
@VSchlattinger Can you confirm this is fine and does not break any other code in the quiz creation and quiz conduction? (in particular with respect to performance)